resip/recon/ConversationManager.cxx

```diff
@@ -1223,6 +1226,30 @@ ConversationManager::onTryingNextTarget(AppDialogSetHandle, const SipMessage& ms
    return true;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// Pager Handler ///////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
+
+const char*
+ConversationManager::sendMessage(const NameAddr& destination, const Data& msg, const Mime& mimeType)
+{
+   if(!mUserAgent->mDum.getMasterProfile()->isMethodSupported(MESSAGE))
+   {
+      mUserAgent->mDum.getMasterProfile()->addSupportedMethod(MESSAGE);
+   }
+   if(!mUserAgent->mDum.getMasterProfile()->isMimeTypeSupported(MESSAGE, mimeType))
+   {
+      mUserAgent->mDum.getMasterProfile()->addSupportedMimeType(MESSAGE, mimeType);
```

Comment:
we shouldn't do this here in recon, the application developer needs to decide which MIME types they want to support so please remove this